### DISCUSSION TITLE
Error creating screenshot after upgrading framework to .net8

### DISCUSSION THREAD

**Discussion by bgoggan** [dlemstra/Magick.NET#1771] [Help]
After upgrading our test framework to use .net8 we started getting the below error when trying to take the screen shots. I tried installing the different packages: Magick.NET-Q8-x64, Magick.NET-Q8-AnyCPU, etc but still the error remains. Any ideas?

System.TypeInitializationException: 'The type initializer for 'NativeMagickImageCollection' threw an exception.'

DllNotFoundException: Unable to load DLL 'Magick.NET-Q8-x64.Native.dll' or one of its dependencies: The specified module could not be found.

**Comment by dlemstra** [comment]
It looks like the native library is not copied to your output folder?

**Reply by bgoggan** [reply]
Yeah, I'm just not sure how or why. Is there something different I'd have to do for that? I had no issues when the framework was built with .net6

**Reply by bgoggan** [reply]
Ok, my apologies. I deleted all the existing code for what was happening and started over. I think there was some old logic in there trying to access that file, which definitely didn't exist, but I've started over and am no longer getting that error. I'll mark this as closed for now. Thanks.
